curation_concerns-models/app/services/curation_concerns/generic_file_indexing_service.rb

```diff
@@ -3,7 +3,9 @@ class GenericFileIndexingService < ActiveFedora::IndexingService
     def generate_solr_document
       super.tap do |solr_doc|
         solr_doc[Solrizer.solr_name('representative')] = object.representative
+        # Label is the actual file name. It's not editable by the user.
         solr_doc[Solrizer.solr_name('label')] = object.label
```

Comment:
I assume someone wanted to be able to search for part of a filename and get a match. That is why it is indexed with `_tesim`, so I preserved that behavior.